### PR TITLE
Kai/igc upload

### PR DESCRIPTION
**Pull request by KaiWissel**

### CODE REVIEW

File: Backend/controller/FlightController.js [hunk 1]
```diff
@@ -48,61 +48,91 @@ router.delete("/:id", async (req, res) => {
...
+      res.status(BAD_REQUEST).send(error);
+    });
+});
+
+// @desc Adds futher data to a existing flight and starts the igc calculation if no igc result are present.
+// @route PUT /flight/:id
+
+router.put("/:id", async (req, res) => {
+  const report = req.body.report;
+  const glider = req.body.glider;
```
Comment (JEK58) [inline Backend/controller/FlightController.js:101]: Glider is a UUID or a string?
Maybe needs to be checked to be a glider of the user.

File: Backend/model/Flight.js [hunk 2]
```diff
@@ -27,14 +44,12 @@ const Flight = db.sequelize.define("Flight", {
   igcUrl: {
     type: DataTypes.STRING,
   },
+  imagesUrls: {
+    type: DataTypes.ARRAY(DataTypes.STRING),
+  },
   glider: {
     type: DataTypes.STRING,
```
Comment (JEK58) [inline Backend/model/Flight.js:51]: Same here maybe a glider object consisting of a UUID and glider name would be better?

**Review by JEK58 — APPROVED** [review]

File: Backend/controller/FlightController.js (hunk 1 above)
Comment (KaiWissel) [reply Backend/controller/FlightController.js:101]: For now it's just a string. Glider has currently no effect on scoring.
Should be a object with brand, name and class in the future.

File: Backend/model/Flight.js (hunk 2 above)
Comment (KaiWissel) [reply Backend/model/Flight.js:51]: s.o.

**Pull request merged**